Modules/Sources/WordPressUI/Extensions/UIImage+Color.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+public extension UIImage {
+
+    static func from(color: UIColor, havingSize size: CGSize = CGSize(width: 1.0, height: 1.0)) -> UIImage {
```

Comment:
(nit) The method signature is a bit odd. I'd go for `UIImage.make(color:size:)` and document that the size Is in points. An initializer could also work.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23471/commits/d910028102fcccdc52cc1f2a1b125fe74625888b and https://github.com/wordpress-mobile/WordPress-iOS/pull/23471/commits/2546931fa9ad6c8703874f63a9a4f4ba8f2562b7